### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jgmac1106**
clarifying the one click install on Reclaim

## Here's what I fixed or added:
disclaimer about one click installing an older version.

## Here's why I did it:
One click install is not available

## Checklist:

- [ ] This pull request addresses a single issue
- [ ] If this code includes interface changes, I've included screenshots in this Pull Request thread
- [ ] I've adhered to [Known's style guide](http://docs.withknown.com/en/latest/developers/standards/) ([these codesniffer rules](http://docs.withknown.com/en/latest/developers/testing/#code-style-testing) might help!)
- [ ] My git branch is named in a descriptive way - i.e., yourname-summary-of-issue
- [ ] I've tested my code in-browser
- [ ] My code contains descriptive comments
- [ ] I've added tests where applicable, and...
- [ ] I can run the [unit tests](http://docs.withknown.com/en/latest/developers/testing/#unit-testing) successfully.

### CODE REVIEW

**Comment by jeremycherfas** [comment]
Took a look at this as I need to update my own, hosted Known. I'm puzzled that in fact there are no Update instructions below, only Installing instructions. Perhaps you could reword? Or add the much-needed Update instructions?

**Comment by jgmac1106** [comment]
Good idea Jeremy I find myself searching the chat logs all the time for update instructions.

File: README.md [hunk 1]
```diff
@@ -8,10 +8,10 @@
 
 ## Installation 
 
-### One-click Known sites
+### Shared Host on  Known sites
 
-If you want to install on your own web space, we recommend [Reclaim Hosting](https://portal.reclaimhosting.com/aff.php?aff=013),
-which includes one-click Known installation. Known is also known to work on [DreamHost](https://dreamhost.com), a high-quality
+If you want to install on your own web space, we recommend [Reclaim Hosting](https://portal.reclaimhosting.com/aff.php?aff=013), which includes one-click Known installation.The one click installs an outdated version of 0.9.2 and then requires updating following the intructions below.
```
Comment (benwerd) [inline README.md]:
```suggestion
If you want to install on your own web space, we recommend [Reclaim Hosting](https://portal.reclaimhosting.com/aff.php?aff=013), which includes one-click Known installation. Currently this installs version 0.9.2, which then requires updating using the instructions below.  
```

**Comment by mapkyca** [comment]
@jgmac1106 Where are we wrt Ben's feedback?

File: README.md (hunk 1 above)
Comment (mapkyca) [reply README.md]: Interestingly, GitHub gives me the button to merge these changes, but gives me a permission error when actually attempting to do so

Comment (benwerd) [reply README.md]: I get the same. It's because we don't own @jgmac1106's branch. He needs to merge them!

Comment (mapkyca) [reply README.md]: Bit daft to show the button if you don't have access to perform the action. Basic UX

**Comment by benwerd** [comment]
I'm closing this and will incorporate @jgmac1106's changes into a new README PR with a few other things.

**Pull request closed without merging**